content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,538 +1,517 @@
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem .
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem .
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca-config.json .
-     ```
+- When you SSH to any node, make sure to add the `-A` flag:
+     
+  ```
+  ssh -A 10.0.0.7
+  ```
 
-     Where `<etcd0-ip-address>` corresponds to the public or private IPv4 of `etcd0`.
+- When using sudo on any node, make sure to preserve the environment so SSH
+  forwarding works:
 
-1. Once this is done, run the following on all etcd machines:
-     ```bash
-     cfssl print-defaults csr > config.json
-     sed -i '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' config.json
-     sed -i 's/www\.example\.net/'"$PRIVATE_IP"'/' config.json
-     sed -i 's/example\.net/'"$PEER_NAME"'/' config.json
-     ```
-     ```bash
-     cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=server config.json | cfssljson -bare server
-     cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=peer config.json | cfssljson -bare peer
-     ```
+  ```
+  sudo -E -s
+  ```
 
-     The above will replace the default configuration with your machine's hostname as the peer name, and its IP addresses. Make sure
-     these are correct before generating the certs. If you found an error, reconfigure `config.json` and re-run the `cfssl` commands.
+### Create load balancer for kube-apiserver
 
-This results in the following files: `peer.pem`, `peer-key.pem`, `server.pem`, `server-key.pem`.
+{{< note >}}
+**Note**: There are many configurations for load balancers. The following
+example is only one option. Your cluster requirements may need a 
+different configuration.
+{{< /note >}}
 
-### {{< tabs name="etcd_mode" >}}
-{{% tab name="Choose one..." %}}
-Please select one of the tabs to see installation instructions for the respective way to set up a virtual IP.
-{{% /tab %}}
-{{% tab name="systemd" %}}
-1. First, install etcd binaries:
-     ```bash
-     ETCD_VERSION="v3.1.12" curl -sSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar -xzv --strip-components=1 -C /usr/local/bin/
-     ```
+1. Create a kube-apiserver load balancer with a name that resolves to DNS.
+
+    In a cloud environment you should place your control plane nodes behind a TCP
+    forwarding load balancer. This load balancer distributes traffic to all
+    healthy control plane nodes in its target list. The health check for
+    an apiserver is a TCP check on the port the kube-apiserver listens on 
+    (default value `:6443`).
+
+    It is not recommended to use an IP address directly in a cloud environment.
+
+    The load balancer must be able to communicate with all control plane nodes
+    on the apiserver port. It must also allow incoming traffic on its
+    listening port.
+
+1. Add the control plane nodes to the load balancer, but be aware they
```

Comment:
We should add a section to ensure our connections through the LB don't time out.

Something like 

1. Confirm the connection is working. A timeout confirms some settings need to be adjusted. A connection refused is ok because the API server is not running yet.
```
nc -v <load balancer ip> <port>
# example: nc -v 10.0.0.0 6443
```

Reply:
💯 👍